File: app/src/main/java/com/github/pockethub/android/ui/item/ContributorItem.java
package com.github.pockethub.android.ui.item;

import android.content.res.Resources;
import android.support.annotation.NonNull;
import android.widget.TextView;

import com.github.pockethub.android.R;
import com.github.pockethub.android.util.AvatarLoader;
import com.meisolsson.githubsdk.model.User;

public class ContributorItem extends UserItem {

    public ContributorItem(AvatarLoader avatarLoader, User dataItem) {
        super(avatarLoader, dataItem);
    }

    @Override
    public void bind(@NonNull ViewHolder holder, int position) {
        super.bind(holder, position);
        TextView contributions = holder.getRoot().findViewById(R.id.tv_contributions);

        Resources res = holder.getRoot().getResources();
        String text = res.getString(R.string.contributions, getData().contributions());

        contributions.setText(text);
    }

    @Override
    public int getLayout() {
        return R.layout.contributor_item;
    }
}

File: app/src/main/java/com/github/pockethub/android/ui/item/ContributorItem.kt
package com.github.pockethub.android.ui.item

import android.widget.TextView
import com.github.pockethub.android.R
import com.github.pockethub.android.util.AvatarLoader
import com.meisolsson.githubsdk.model.User
import com.xwray.groupie.kotlinandroidextensions.ViewHolder

class ContributorItem(avatarLoader: AvatarLoader, user: User) : UserItem(avatarLoader, user) {

    override fun getLayout() = R.layout.contributor_item

    override fun bind(holder: ViewHolder, position: Int) {
        super.bind(holder, position)
        val contributions = holder.root.findViewById<TextView>(R.id.tv_contributions)

        val res = holder.root.resources
        val text = res.getString(R.string.contributions, user.contributions())

        contributions.text = text
    }
}
